### ISSUE TITLE
scleroderma renal crisis

### ISSUE THREAD

**Issue by anastasia-coding** [obophenotype/human-phenotype-ontology#11345]
**Preferred term label:**
Scleroderma renal crisis

**Synonyms**
-----

**Definition (free text, please give PubMed ID)**
Scleroderma renal crisis (SRC) is characterised by the abrupt onset of hypertension, acute kidney injury and microangiopathic haemolytic anaemia in the context of systemic sclerosis. (PMID: 39555050)

**Parent term (use hpo.jax.org/app)**
Hypertension HP:0000822

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Limited cutaneous systemic sclerosis (ORPHA:220402)
Diffuse cutaneous systemic sclerosis (ORPHA:220393)
Systemic sclerosis (ORPHA:90291) 
Mixed Connective Tissue Disease (ORPHA:809)

**Your nano-attribution (ORCID)**
Anastasia-Vasiliki Madenidou (https://orcid.org/0000-0002-7382-9958)

**Comment by pnrobinson** [comment]
@anastasia-coding 
This is a bundled manifestation rather than a single phenotypic observation, and could be represented with the underlying HPO terms

Acute kidney injury 
Severe hypertension 
Renal thrombotic microangiopathy 
Proteinuria 
Hematuria
Oliguria
